src/components/PersonLink.tsx

```diff
@@ -0,0 +1,18 @@
+import cn from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person | undefined;
+};
+
+export const PersonLink:React.FC<Props> = ({ person }) => {
+  return (
+    <Link
+      to={`/people/${person?.slug}`}
+      className={cn({ 'has-text-danger': person?.sex === 'f' })}
```

Comment:
It would be very cool to put such strings into separate variables to avoid using magic data.
For example:
```
const FEMALE = 'f';
const NOT_SET_VALUE = '-';
```
This applies to all code.